daemon/config.go

```diff
@@ -38,6 +38,7 @@ type CommonConfig struct {
 	// multihost networking (to store networks and endpoints information) and by the node discovery
 	// mechanism.
 	ClusterStore string
+	ClusterOpts  map[string]string
```

Comment:
Can we please comment the exported fields? We're trying to make this a habit :-)